packages/calcite-components/src/components/slider/slider.e2e.ts

```diff
@@ -108,1219 +112,1351 @@ describe("calcite-slider", () => {
+          const isMinThumbFocused = await isElementFocused(page, `.${CSS.thumbMinValue}`, { shadowed: true });
 
-        expect(isMinThumbFocused).toBe(true);
-        await assertValuesUnchanged(expectedValue);
-      });
+          expect(isMinThumbFocused).toBe(true);
+          await assertValuesUnchanged(expectedValue);
+        });
 
-      it("mirrored: click/tap should grab the max value thumb", async () => {
-        await setUpTest(`${commonSliderAttrs} min-value="${expectedValue}" max-value="${expectedValue}" mirrored`);
+        it("mirrored: click/tap should grab the max value thumb", async () => {
+          await setUpTest(
+            `${commonSliderAttrs} min-value="${expectedValue}" max-value="${expectedValue}" mirrored layout=${layout}`,
+          );
 
-        await assertValuesUnchanged(expectedValue);
+          await assertValuesUnchanged(expectedValue);
 
-        await page.mouse.click(trackRect.x, trackRect.y);
-        await page.waitForChanges();
+          layout === "horizontal"
+            ? await page.mouse.click(trackRect.x, trackRect.y)
+            : await page.mouse.click(trackRect.x, trackRect.y + trackRect.height);
+          await page.waitForChanges();
 
-        const isMinThumbFocused = await isElementFocused(page, `.${CSS.thumbMinValue}`, { shadowed: true });
+          const isMinThumbFocused = await isElementFocused(page, `.${CSS.thumbMinValue}`, { shadowed: true });
 
-        expect(isMinThumbFocused).toBe(true);
-        await assertValuesUnchanged(expectedValue);
+          expect(isMinThumbFocused).toBe(true);
+          await assertValuesUnchanged(expectedValue);
+        });
       });
     });
-  });
 
-  describe("when a range has 0 for both minValue and maxValue", () => {
-    const slider = `<calcite-slider
-      min="-10"
-      max="1"
-      min-value="0"
-      max-value="0"`;
-    const nonMirroredSlider = `<div style="width: 300px; margin: 1rem;">${slider}></calcite-slider></div>`;
-    const mirroredSlider = `<div style="width: 300px; margin: 1rem;">${slider} mirrored></calcite-slider></div>`;
-
-    it("should position the minValue thumb beside the maxValue thumb", async () => {
-      const page = await newE2EPage({ html: nonMirroredSlider });
-      const minValueThumb = await page.find("calcite-slider >>> .thumb--minValue");
-      const maxValueThumb = await page.find("calcite-slider >>> .thumb--value");
-      const minHandleLeft = await (await minValueThumb.getComputedStyle()).left;
-      const maxHandleRight = await (await maxValueThumb.getComputedStyle()).right;
-      expect(minHandleLeft).toBe("260px");
-      expect(maxHandleRight).toBe("26px");
-    });
+    describe("when a range has 0 for both minValue and maxValue", () => {
+      const slider = `<calcite-slider
+        min="-10"
+        max="1"
+        min-value="0"
+        max-value="0"
+        layout={layout}`;
```

Comment:
Shouldn't this be `layout=${layout}` to use the `layout` var from the `describe.each` block? This would eliminate the need to use `replace` below and keeps tests consistent.